api/turing/api/ensembling_job_api.go

```diff
@@ -109,6 +109,47 @@ func (c EnsemblingJobController) ListEnsemblingJobs(
 	return Ok(results)
 }
 
+type deleteEnsemblingJobResponse struct {
+	ID models.ID `schema:"id"`
+}
+
+// DeleteEnsemblingJob deletes and ensembling job and cancels any ongoing process.
+func (c EnsemblingJobController) DeleteEnsemblingJob(
+	_ *http.Request,
+	vars RequestVars,
+	_ interface{},
+) *Response {
+	options := &GetEnsemblingJobOptions{}
+
+	if err := c.ParseVars(options, vars); err != nil {
+		return BadRequest(
+			"failed to fetch ensembling job",
+			fmt.Sprintf("failed to parse query string: %s", err),
+		)
+	}
+
+	ensemblingJob, err := c.EnsemblingJobService.FindByID(
+		*options.ID,
+		service.EnsemblingJobFindByIDOptions{
+			ProjectID: options.ProjectID,
+		},
+	)
+	if err != nil {
+		return NotFound("ensembler not found", err.Error())
```

Comment:
This message is confusing. Shouldn't it be `Job not found`?

Reply:
ok